ngraph/test/backend/erf.in.cpp

```diff
@@ -1,52 +0,0 @@
-// Copyright (C) 2018-2021 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
```

Comment:
why is this test removed? it is useful for different backend implementations.

Reply:
Just forwarded you the email. I think template plugin would be used to validate the reference OP.  CPU plugin will use SLT to validate OP tests.